apps/avifenc.c

```diff
@@ -773,19 +773,6 @@ int main(int argc, char * argv[])
             }
         } else if (!strcmp(arg, "-l") || !strcmp(arg, "--lossless")) {
             lossless = AVIF_TRUE;
-
-            // Set defaults, and warn later on if anything looks incorrect
```

Comment:
The new code only allows correct parameters to be used with lossless. It seems that this pull request should also remove the code that the comment "warn later on if anything looks incorrect" refers to. I took a quick look and it seems more complicated than I thought because we sometimes set `lossless` to `AVIF_TRUE`.

Reply:
Indeed, I'd rather do it in a different pull request.